server/controller/user.js

```diff
@@ -0,0 +1,60 @@
+import db from '../models';
+
+/**
+* Class representing controller
+*
+* @class userController
+*/
+class userController {
+  /**
+  * Register a user on the platform
+  *
+  * @static
+  * @param {object} req - The request object
+  * @param {object} res - The response object
+  * @return {object} Success message with the user created or error message
+  * @memberof userController
+  */
+  static signup(req, res) {
+    return db.User
+      .create({
+        firstname: req.body.firstname,
+        lastname: req.body.lastname,
+        email: req.body.email,
+        password: req.body.password,
+        isAdmin: req.body.isAdmin
+      })
+      .then(user => res.status(201).json({
+        status: 'success',
+        message: `User with userId: ${user.id} was successfully signed up`,
+
+      }))
+      .catch((error) => {
+        for (let i = 0; i < error.errors.length; i += 1) {
+          const errorHandler = error.errors[i].message;
+          switch (errorHandler) {
```

Comment:
☺️ what are you doing man

Reply:
I was trying to handle the sequelize validation errors